host/CMakeLists.txt

```diff
@@ -137,8 +136,15 @@ if (OE_SGX)
   set(PLATFORM_FLAGS "-m64")
 elseif(OE_TRUSTZONE)
   list(APPEND PLATFORM_SRC
-    optee/enclave.c
     optee/log.c)
+  
+  if (UNIX)
+    list(APPEND PLATFORM_SRC
+      optee/linux/enclave.c)
+  else()
+    message(STATUS "Add Windows OP-TEE files.")
```

Comment:
My only suggestion would be to swap this to an error.

Reply:
Will do.